src/lib/cmdlinergen.ml

```diff
@@ -175,4 +176,7 @@ module Gen () = struct
       , Cmdliner.Term.info wire_name ~doc )
     in
     terms := generate :: !terms
+
+  let declare name desc_list ty = declare_ true name desc_list ty
```

Comment:
I think this should be false and the one below true. It would be good to add a test for it somewhere or extend an example